src/Uno.Extensions.Maui.UI/Interop/MauiInterop.cs

```diff
@@ -5,18 +5,18 @@ internal static class MauiInterop
 	private static readonly List<ControlMapping> _mappings = new();
 	private static readonly List<IWinUIToMauiStyleHandler> _styleHandlers = new();
 
-	static MauiInterop()
-	{
-		MapControl<TextBlock, Microsoft.Maui.Controls.Label>();
-		MapControl<TextBox, Microsoft.Maui.Controls.Entry>();
-		MapControl<Grid, Microsoft.Maui.Controls.Grid>();
-		MapControl<StackPanel, Microsoft.Maui.Controls.StackLayout>();
-
-		MapStyleHandler<LabelStyleHandler>();
-		MapStyleHandler<VisualElementStyleHandler>();
-		MapStyleHandler<ViewStyleHandler>();
-		MapStyleHandler<LayoutStyleHandler>();
-	}
+	// static MauiInterop()
```

Comment:
Can we remove this?

Reply:
Yes, Dan mentioned that should be worked, and commenting on this code, for now, will prevent that we add any behavior to the controls. Of course, we can delete it and find it on the git history later, but I would say that would add more work than leaving the code commented.